FreeRTOS/FreeRTOS
# Update riscv eclipse project

Description
-----------
Exclude FreeRTOS/FreeRTOS/Source/examples folder in following eclipse projects:
* RISC-V_RV32M1_Vega_GCC_Eclipse
* RISC-V_RV32_SiFive_HiFive1-RevB_FreedomStudio
* RISC-V_RV64_PolarFire_SoftConsole

Test Steps
-----------
Before this PR, there will be `xTaskCreateStatic` undefined reference error.
```
FreeRTOS/FreeRTOS/Demo/RISC-V_RV32M1_Vega_GCC_Eclipse/projects/RTOSDemo_ri5cy/debug/../../../../../Source/examples/cmake_example/main.c:47: undefined reference to `xTaskCreateStatic'
```

Checklist:
----------



Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
